lms/djangoapps/verify_student/models.py

```diff
@@ -167,6 +167,21 @@ def should_display_status_to_user(self):
         """
         return False
 
+    @property
+    def expiration_datetime(self):
+        """
+        [UCSD_CUSTOM] Datetime that the verification will expire.
```

Comment:
```suggestion
        Datetime that the verification will expire.
```

Reply:
It was decided that we should add comment tags like this so that it would be easy to identify and search the custom changes to Open edX codebase.